packages/webapp/src/components/ui/button/Button.tsx

```diff
@@ -4,6 +4,8 @@ import { cva } from 'class-variance-authority';
 import { forwardRef } from 'react';
 import { cn } from '../../../utils/utils';
 
+export type ButtonVariants = 'primary' | 'secondary' | 'success' | 'danger' | 'zombie' | 'zombieGray' | 'yellow' | 'black' | 'active' | 'hover' | 'zinc';
```

Comment:
to avoid desync you can maybe do something like `export type ButtonVariants = VariantProps<typeof buttonStyles>['variant'];`

Reply:
Thanks, updated